docs/docs/build/dashboards/customization.md

```diff
@@ -130,22 +132,22 @@ For more details about configuring themes, you can refer to our [Theme YAML](/re
 ```yaml
 type: explore
 
-title: Title of your explore dashboard
-description: a description
+title: "Title of your explore dashboard"
+description: "The description of your dashboard"
 metrics_view: <your-metric-view-file-name>
 
-dimensions: '*' #can use expressions
-measures: '*' #can use expressions
+dimensions: '*' # can use expressions
+measures: '*' # can use expressions
 
-theme: #your default theme
+theme: # your default theme
 
-time_ranges: #was available_time_ranges
-time_zones: #was available_time_zones
+time_ranges: # was available_time_ranges
+time_zones: # was available_time_zones
 
-defaults: #define all the defaults within here
+defaults: # define all the defaults within here
     dimensions:
```

Comment:
This isn't really an example, since most of the property values are missing. Mind improving this?